### PR TITLE
Feature/num 21 form 

### PR DESCRIPTION
**Pull request by ErickS09**
## Description

The registry section was created using the antdesing inputs.

## Trello

Include NUM-21

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: src/pages/Home.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import HeroSection from '../sections/home/HeroSection/HeroSection'
 import { Button, Icon } from '../components'
 import ActivitiesSection from '../sections/home/ActivitiesSection/ActivitiesSection'
+import FormSection from '../sections/home/FormSection/FormSection'
```
Comment (Georgge) [inline src/pages/Home.js]: @ErickS09  Esto lo comentamos en la ultima llamada, no debe de llamarse FormSection sino RegisterSection

File: src/sections/home/FormSection/FormSection.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        console.log('Error:', error)
+    }
+
+    return (
+        <div className={styles.container}>
+            <div className={styles.text}>
+                <div className={styles.title}>No lo olvides, registrate hoy y recibe un <span className={styles.desc}>30% de descuento</span> en tu primer viaje.
+                </div>
+            </div>
+
```
Comment (Georgge) [inline src/sections/home/FormSection/FormSection.js]: Estos espacios estan de más, hay que quitarlos

File: src/sections/home/FormSection/formsection.module.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+.form{
+    height: 44px;
+    width: 332px;
+    margin-left: 46px;
+    border-radius: 6px;
+    align-items: center;
+}
+
```
Comment (Georgge) [inline src/sections/home/FormSection/formsection.module.css]: Estos dobles espacios SI estan bien. En css podemos seguir trabajando asi

File: src/sections/home/HeroSection/herosection.module.css [hunk 4]
```diff
@@ -43,7 +43,7 @@
 .navbar{
     padding-right: 75px;
     border-bottom: 1px solid white;
-    flex-grow: 1;
+    flex-grow: 1px;
```
Comment (Georgge) [inline src/sections/home/HeroSection/herosection.module.css]: @ErickS09 lee más sobre flexbox y css. Esto deberia de ser un valor numerico no en px. Deja solo el 1

**Review by nthchild1 — APPROVED** [review]

**Pull request merged**